README.md

```diff
@@ -27,9 +24,15 @@ Merge conflicts happen when two people make changes to the same file on GitHub
 - **What you'll learn**: What merge conflicts are, how you resolve merge conflicts, how to reduce merge conflicts.
 - **What you'll build**: We'll work with a short Markdown resume file in this course.
 - **Prerequisites**: We recommend taking [Introduction to GitHub](https://github.com/skills/introduction-to-github) prior to this course.
-- **How long**: This course is three steps long and takes less than 30 minutes to complete.
+- **How long**: This course  takes less than 30 minutes to complete.
```

Comment:
```suggestion
- **How long**: This course takes less than 30 minutes to complete.
```

Reply:
I am happy to see that the coarse is still working and is still only 30 minutes long and easy to understand.